README.md

```diff
@@ -67,19 +67,23 @@ Disables the current custom theme
 #### `updateTheme(colorObject)`
 Updates the colors used in the custom theme and redraws the theme.
 
-- colorObject - `Object` -  A colors object containing `backPrimary`, `backSecondary`,
+- colorObject - `Object` -  A colors object containing `type`, `backPrimary`, `backSecondary`,
 `backHighlight`, `forePrimary` and `foreSecondary` attributes any attribute not
 included will not be changed.
 
+The `type` attribute can be any value from `window.GMusicTheme.TYPES`.  At the moment
+these are `"FULL"` and `"HIGHLIGHT_ONLY"`.  There are constants for these values that
+can be accessed through the types object.  E.g. `window.GMusicTheme.TYPES.HIGHLIGHT_ONLY`
```

Comment:
Not required but it might be nice to provide a UI preview of what "highlight" mode looks like as opposed to "full"